apps/console/src/features/users/components/guests/api/invite.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { AsgardeoSPAClient, HttpClientInstance } from "@asgardeo/auth-react";
+import { HttpMethods } from "@wso2is/core/models";
+import { AxiosError, AxiosRequestConfig, AxiosResponse } from "axios";
+import useRequest, { RequestErrorInterface, RequestResultInterface } from "../../../../core/hooks/use-request";
+import { store } from "../../../../core/store";
+import { InvitationsInterface, UserInviteInterface } from "../models/invite";
+
+/**
+ * Initialize an axios Http client.
+ *
+ */
+const httpClient: HttpClientInstance = AsgardeoSPAClient.getInstance().httpRequest.bind(
+    AsgardeoSPAClient.getInstance());
+
+/**
+ * Hook to get the parent org user invites list.
+ */
+export const useGetParentOrgUserInvites = <Data = InvitationsInterface,
+    Error = RequestErrorInterface>(): RequestResultInterface<Data, Error> => {
+
+    const requestConfig: AxiosRequestConfig = {
+        headers: {
+            "Content-Type": "application/json"
+        },
+        method: HttpMethods.GET,
+        url: store.getState().config.endpoints.guestsList
+    };
+
+    const { data, error, isValidating, mutate } = useRequest<Data, Error>(requestConfig);
+
+    return {
+        data,
+        error: error,
+        isLoading: !error && !data,
+        isValidating,
+        mutate: mutate
+    };
+};
```

Comment:
Move GET request to a new file
ref: https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/branding/api/use-get-custom-text-preference-fallbacks.ts